sdk/storage/azblob/ChangeLog.md

```diff
@@ -0,0 +1 @@
+# Change Log
```

Comment:
This should be filled out indicating this is the initial release. also I believe it should be CHANGELOG, verify with azcore please

Reply:
We should also name this file `CHANGELOG.md` for consistency with everything else.

Reply:
Done.